enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -24,25 +25,48 @@ Features currently supported:
 - ability for the user to modify existing questions
 - theming with CSS variables
 - plugins for custom actions, overriding dashboard card menu items
-- subscribing to events
-- editing dashboards - requires upgrade to metabase v50
-- creating dashboards
-
-Features not yet supported:
-
-- letting users create new questions from scratch
+- subscribing to user or system events
+- creating and editing dashboards
+- create new questions from scratch and modifying existing questions
+
+## Known limitations
+
+* The SDK is currently only compatible with Metabase v50
+* Some of the Pro/EE features are not exposed in the UI
+  * Verified content
+  * Official collections
+  * Subscriptions
+  * Alerts
+  * ...
+* The Metabase Embedding SDK does not support server-side rendering (SSR) at the moment.
+  * If you are using a framework with SSR support such as Next.js or Remix, you have to ensure that the SDK components are rendered on the client side.
+  * For example, you can apply the `"use client"` directive on Next.js or use the `remix-utils/ClientOnly` component
+      on Remix.
```

Comment:
Let's remove these two lines until we can confirm that it works correctly in Next.js and Remix.

Reply:
OK